### PR TITLE
Adds meat donuts

### PR DESCRIPTION
**Pull request by Supermichael777**
:cl:
add: NTs crack archeo-gastronomers have discovered a long lost recipe for a meat based donut.
/:cl:

You thought it was delicious puff pastry. BUT IT WAS I, MEATO, who entered your lips.

### CODE REVIEW

**Comment by Kagus** [comment]
Pfeh. Keep this kind of nonsense up, and next thing the lizards will start to actually feel _welcome_ on the station security force.

**Comment by Supermichael777** [comment]
fun fact sec officers can eat any doughnut without penalty.

**Comment by ExcessiveUseOfCobblestone** [comment]
food gore shouldn't be tolerated on my pious codebase

**Comment by epochayur** [comment]
🤢

**Comment by fen0s** [comment]
please no

**Comment by MrDoomBringer** [comment]
what @81Denton said

**Comment by Doppeltongue** [comment]
But why?

File: code/modules/food_and_drinks/recipes/tablecraft/recipes_pastry.dm [hunk 1]
```diff
@@ -22,6 +22,16 @@
 	)
 	result = /obj/item/reagent_containers/food/snacks/donut
 	subcategory = CAT_PASTRY
+	
+datum/crafting_recipe/food/donut
```
Comment (octareenroon91) [inline code/modules/food_and_drinks/recipes/tablecraft/recipes_pastry.dm]: duplicate typepath

**Review by octareenroon91 — CHANGES_REQUESTED** [review]

**Comment by WJohn** [comment]
that's fucking disgusting

**Comment by Arkvold** [comment]
... I'm simultaneously sickened and intrigued.  It would be _slightly_ less disgusting if it used actually-cooked meat.

I do appreciate that this establishes that Security will eat damn near anything so long as it's vaguely donut-esque.

Can you arrange for these to spawn in donut boxes?  Or will that happen in the existing configuration, since they're a child of food/snacks/donut?

**Comment by zamolxius** [comment]
hot

**Comment by Potato-Masher** [comment]
5% chance that donuts in normal boxes that spawn are replaced by a meat donut.

**Review by ninjanomnom — APPROVED** [review]
The food gore shadow council has my family. I'm sorry but I must merge this.

**Pull request merged**